tools/flash-shelly.py

```diff
@@ -227,7 +228,7 @@ def add_service(self, zc, service_type, device):
       logger.debug(f"[Device Scan] found device: {host}, IP address: {wifi_ip}")
       logger.trace(f"[Device Scan] info: {info}")
       logger.trace(f"[Device Scan] properties: {properties}")
-      if properties.get('arch') == 'esp8266':  # this detects if esp device.
+      if properties.get('arch') == 'esp8266' or properties.get('arch') == 'esp32':  # this detects if esp device.
```

Comment:
maybe use `if properties.get('arch') in ('esp8266', 'esp32'):`, slightly cleaner.